docs/framework/data/adonet/dataset-datatable-dataview/security-guidance.md

```diff
@@ -69,7 +69,7 @@ The object type restrictions don't apply when using `DataAdapter.Fill`, such as
 
 An app can extend the allowed types list to include custom types in addition to the built-in types listed above. If extending the allowed types list, the change affects _all_ `DataSet` and `DataTable` instances within the app. Types cannot be removed from the built-in allowed types list.
 
-#### Extend through configuration (.NET Framework 4.0 - 4.8)
+#### Extend through configuration (.NET Framework 4.0 - 4.8.1)
```

Comment:
I'm wondering if we should move to "and later", like is used for .NET Core. That would get us out of the business of needing to update this again if/when we have 4.8.2 :) I think this is fine, but I wanted to bring it up as an opportunity to reduce future work around these docs.